assets/src/components/PostMeta/withPostMeta.js

```diff
@@ -21,51 +33,35 @@ export function withPostMeta( WrappedComponent ) {
       super( props );
       this.handleChange = this.handleChange.bind( this );
       this.valuePropName = getValuePropName( WrappedComponent );
+      this.state = {
+        value: null
```

Comment:
Still not sure if we need to introduce state and `getDerivedStateFromProps` in the component to do handle translating the props into a value. What is the benefit over calling `getValueFromProps` in render?

Reply:
This is not needed anymore as you removed the `componentDidUpdate` method in your previous commit.